src/utils.js

```diff
@@ -127,25 +127,62 @@ exports.hideAbsolutePathsInObject = (object) => {
   }
 
   if (_.isString(object)) {
-    const words = object.split(' ').map(word => (path.isAbsolute(word) ? path.basename(word) : word))
-    return words.join(' ')
+    const words = object.split(' ')
+    .map(word => (path.isAbsolute(word) ? path.basename(word) : word))
+    .join(' ')
+    const anonymizedWords = exports.anonymizePaths(words)
+    return anonymizedWords
   }
 
   // Avoid "Maximum call stack size exceeded" errors
   object = jc.decycle(object)
 
-  return _.deepMapValues(object, (value) => {
-    if (!_.isString(value)) {
-      return value
+  return _.deepMapValues(object, exports.anonymizePaths)
+}
+
+/**
+ * @summary Anonymize paths in logs
+ * @function
+ * @public
+ *
+ * @param {String|Object} value - value to anonymize
+ * @returns {String|Object} transformed value
+ */
+exports.anonymizePaths = (value) => {
+  if (!_.isString(value)) {
+    return value
+  }
+  // Don't alter disk devices, even though they appear as full paths
+  if (value.startsWith('/dev/') || value.startsWith('\\\\.\\')) {
+    return value
+  }
+
+  const valueArray = value.split('\n')
```

Comment:
`lines` would be a better name for `valueArray`.

Reply:
Yes, I had actually named it `lines` while developing but then renamed it back to this while refactoring, changing it now so it's clearer